### ISSUE TITLE
GSM

### ISSUE THREAD

**Issue by felixker** [steve-m/kalibrate-rtl#29]
I don't consider this a bug.

Singapore doesn't use GSM900/850. Does it mean this program can't be used on my end?

**Comment by ValZapod** [comment]
This is not true. https://www.frequencycheck.com/countries/singapore It does support E-GSM and 1800 (DCS) which our kal does support. https://github.com/steve-m/kalibrate-rtl/blob/aae11c8a8dc79692a94ccfee39ba01e8c8c05d38/src/arfcn_freq.cc#L53
